icq-relayer/cmd/root.go

```diff
@@ -1,11 +1,21 @@
 package cmd
 
 import (
+	"errors"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/config"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/keys"
+	"github.com/cosmos/cosmos-sdk/server"
+	svrcmd "github.com/cosmos/cosmos-sdk/server/cmd"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/quicksilver-zone/quicksilver/app"
+	cmdcfg "github.com/quicksilver-zone/quicksilver/cmd/config"
+	"github.com/spf13/viper"
+	tmcfg "github.com/tendermint/tendermint/config"
```

Comment:
Please separate standard library imports from the other imports and perhaps please place imports into these blocks, each separated by a new line.

Block 1. Standard library imports

Block 2. Imports not from this package/library

Block 3. Imports from this package/library

Reply:
thank you for review, i just updated